conf.d/done.fish

```diff
@@ -175,7 +175,7 @@ if test -z "$SSH_CLIENT" # not over ssh
                 if test $exit_status -ne 0
                     set urgency "--urgency=critical"
                 end
-                notify-send $urgency --icon=terminal --app-name=fish "$title" "$message"
+                notify-send -u $urgency --icon=terminal --app-name=fish "$title" "$message"
```

Comment:
Needs to account for empty $urgency variable

Reply:
Also, favour `--urgency=` to `-u` for readability and consistency

Reply:
Done for `--urgency=`, regarding the default value, is taken into account in the previous if statement.

Reply:
Point is that there might not be a urgency level defined by the user, in which case the it won't work if the command finishes with a 0 exit status

Reply:
The latest commit resolve also this situation